composer.json

```diff
@@ -20,20 +20,22 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "minimum-stability": "dev",
```

Comment:
I know, but while rather remote because of the "prefer-stable" flag, it looked like it might cause other versions to be pulled in on travis tests on this repo as opposed to what will be pulled in on project installs